server/locale/DE/Pi30/mutations/pi30.js

```diff
@@ -0,0 +1,205 @@
+import Logo from '../../../../message/logos';
+import {
+    xSmallFallback,
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallNoWrap,
+    addPeriod
+} from '../../../../message/mediaQueries';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['kaufen,']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['30']
+    }
+].reduce((acc, item) => {
+    const { sizes, breaks } = item;
+    sizes.forEach(size => {
+        acc.push({
+            tag: size,
+            br: breaks
+        });
+    });
+    return acc;
+}, []);
+
+const flex = [
+    [
+        'default',
+        {
+            logo: Logo.PP_PAYPAL.WHITE,
+            headline: [
+                {
+                    tag: 'xsmall'
+                },
+                {
+                    tag: 'medium'
+                }
+            ],
+            disclaimer: ['default']
+        }
+    ],
```

Comment:
```suggestion
    ],
        [
        'ratio:20x1',
        {
            styles: [addPeriod()],
            headline: [...headlineBreaks]
        }
    ],
```
Let's add 20x1 into this file so that we can make sure to account for the missing period via the `addPeriod()` media query.